homeassistant/components/blink/sensor.py

```diff
@@ -1,25 +1,29 @@
 """Support for Blink system camera sensors."""
 import logging
 
-from homeassistant.const import CONF_MONITORED_CONDITIONS
+from homeassistant.const import TEMP_FAHRENHEIT
 from homeassistant.helpers.entity import Entity
 
-from . import BLINK_DATA, SENSORS
+from .const import DOMAIN, TYPE_BATTERY, TYPE_TEMPERATURE, TYPE_WIFI_STRENGTH
 
 _LOGGER = logging.getLogger(__name__)
 
+SENSORS = {
+    TYPE_TEMPERATURE: ["Temperature", TEMP_FAHRENHEIT, "mdi:thermometer"],
+    TYPE_BATTERY: ["Battery", "", "mdi:battery-80"],
```

Comment:
We should preferably use device classes instead and remove custom icons.

https://developers.home-assistant.io/docs/entity_sensor#available-device-classes

Reply:
It's ok to do in a future PR. :+1: